snap/component.go

```diff
@@ -70,29 +70,33 @@ func (csi *ComponentSideInfo) Equal(other *ComponentSideInfo) bool {
 	return *csi == *other
 }
 
+// ComponentBaseDir returns where components are to be found for the
+// snap with name instanceName.
+func ComponentsBaseDir(instanceName string) string {
+	return filepath.Join(BaseDir(instanceName), "components")
```

Comment:
Is "components" a reserved snap name both in the store and in snapd itself?

Reply:
I don't know, but it is under instance name so it won't conflict.